source/deployment/bulk-loading-data.rst

```diff
@@ -0,0 +1,25 @@
+.. _bulk-loading-data:
+
+Running the bulk loading command
+================================
+
+Before running the bulk loading command, you must first create a JSON file that contains the data that you want to import. After you create the file, run the bulk load command in validation mode. In this mode, the data is checked for correctness, but is not written to the database. After validating, run the command in apply mode, which saves the data to the database.
```

Comment:
My understanding is that for those who do not have a firm grasp of English, having "that" makes the sentence easier to parse and understand. It's not grammatically incorrect to have "that", so I think we should keep it here.